README.md

```diff
@@ -176,9 +176,9 @@ Modify the downloaded TensorFlow script to run TensorFlow with nGraph optimizati
 
     import ngraph_bridge
     ...
-    config = tf.ConfigProto()
+    config = tf.compat.v1.ConfigProto()
```

Comment:
The default is 1.15? will these apis work with 1.15?

Reply:
They are working. All CI pipelines are running with 1.15 and passing.